src/ui.form-date-picker-range/configs/default.config.js

```diff
@@ -44,6 +44,11 @@ export default /*tw*/ {
   periodDateYearList: "grid grid-cols-3 grid-rows-1 gap-0.5",
   periodDateQuarterList: "",
   periodDate: "w-full",
+  periodDateInRange: "bg-brand-100 text-brand-900 hover:!bg-brand-200 rounded-none",
```

Comment:
Should be general key with same styles for button and specific keys for each case to aviod styling hell in future :)

Reply:
`periodDate` is general key. This one and others are specific keys.